packages/orbit-components/src/BadgeList/BadgeListItem/index.js.flow

```diff
@@ -17,8 +17,8 @@ export type Props = {|
 |};
 
 declare export var getIconColor: (type: Type) => string;
-declare export var StyledBadgeListItem: StyledComponent<any, any, HTMLDivElement>;
-declare export var StyledVerticalBadge: StyledComponent<any, any, HTMLDivElement>;
-declare export var StyledBadgeContent: StyledComponent<any, any, HTMLDivElement>;
+declare export var ItemWrapper: StyledComponent<any, any, HTMLDivElement>;
+declare export var VerticalBadge: StyledComponent<any, any, HTMLDivElement>;
+declare export var BadgeContent: StyledComponent<any, any, HTMLDivElement>;
```

Comment:
These can be removed. We're not exporting them to the outside and we don't use flow internally (and we're even removing them from Orbit entirely soon™) so I'd just remove them now.